### PR TITLE
virsh_list: Update set remote_ip/local_ip params

### PR DESCRIPTION
**Pull request by cliping**

### CODE REVIEW

**Comment by cliping** [comment]
Test result：

#avocado run --vt-type libvirt --vt-machine-type i440fx virsh.list.special_test.remote.with_valid_options.list_table.all,virsh.list.special_test.remote.with_valid_options.list_table.inactive,virsh.list.special_test.remote.with_valid_options.list_table.none,virsh.list.special_test.remote.with_valid_options.list_name.inactive,virsh.list.special_test.remote.with_valid_options.list_name.all,virsh.list.special_test.remote.with_valid_options.list_name.none,virsh.list.special_test.remote.with_valid_options.list_uuid.inactive,virsh.list.special_test.remote.with_valid_options.list_uuid.all,virsh.list.special_test.remote.with_valid_options.list_uuid.none,virsh.list.special_test.remote.with_valid_options.list_none.inactive,virsh.list.special_test.remote.with_valid_options.list_none.all,virsh.list.special_test.remote.with_valid_options.list_none.none,virsh.list.special_test.remote.with_valid_options.list_title.inactive,virsh.list.special_test.remote.with_valid_options.list_title.all,virsh.list.special_test.remote.with_valid_options.list_title.none
JOB ID     : ec9bc619fbd6857c4dc9b7f2a260207f7ca166eb
JOB LOG    : /root/avocado/job-results/job-2017-08-28T03.47-ec9bc61/job.log
 (01/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_table.inactive: PASS (113.94 s)
 (02/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_table.all: PASS (103.23 s)
 (03/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_table.none: PASS (116.25 s)
 (04/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_name.inactive: PASS (113.68 s)
 (05/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_name.all: PASS (113.13 s)
 (06/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_name.none: PASS (112.40 s)
 (07/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_uuid.inactive: PASS (114.05 s)
 (08/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_uuid.all: PASS (111.68 s)
 (09/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_uuid.none: PASS (112.62 s)
 (10/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_none.inactive: PASS (113.21 s)
 (11/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_none.all: PASS (111.74 s)
 (12/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_none.none: PASS (114.20 s)
 (13/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_title.inactive: PASS (115.44 s)
 (14/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_title.all: PASS (98.73 s)
 (15/15) type_specific.io-github-autotest-libvirt.virsh.list.special_test.remote.with_valid_options.list_title.none: PASS (99.19 s)
RESULTS    : PASS 15 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1670.98 s

File: libvirt/tests/src/virsh_cmd/monitor/virsh_list.py [hunk 1]
```diff
@@ -69,12 +69,12 @@ def list_local_domains_on_remote(options_ref, remote_ip, remote_passwd,
...
     remote_ref = params.get("remote_ref", "")
-    remote_ip = params.get("remote_ip", "REMOTE.EXAMPLE.COM")
+    remote_ip = params.get("remote_ip")
     remote_pwd = params.get("remote_pwd", None)
-    local_ip = params.get("local_ip", "LOCAL.EXAMPLE.COM")
+    local_ip = params.get("local_ip")
     remote_user = params.get("remote_user", "root")
     local_user = params.get("username", "root")
-    local_pwd = params.get("local_pwd", "password")
+    local_pwd = params.get("local_pwd", None)
```
Comment (waynesun09) [inline libvirt/tests/src/virsh_cmd/monitor/virsh_list.py]: pls check the EXAMPLE parameter and raise error if not replaced by user

Comment (cliping) [reply libvirt/tests/src/virsh_cmd/monitor/virsh_list.py]: @waynesun09 , updated code, please help review again, thanks.

**Comment by waynesun09** [comment]
ACK and merge now, thanks

**Pull request merged**